### DISCUSSION TITLE
Rework Suggestions for Silencer - Adding Complexity and Depth to Gameplay

### DISCUSSION THREAD

**Discussion by etamimi93** [ValveSoftware/Dota2-Gameplay#15439] [Ideas]
It goes without saying that the numbers mentioned herein need testing and optimization. Still, I opted to include them as a base or starting point to evaluate the feasibility of the proposed changes. I used to be a Silencer spammer (over 2k games) but I stopped playing him because his kit has been stagnant over the last couple of years. I aim to explore the feasibility of raising the complexity level of the hero's gameplay without placing an undue burden on development, balancing, and optimization.

### Arcane Curse (Rework)

- No longer deals additional slow and damage to the silenced target. Base and penalty durations are unchanged.

- Damage per second increased to 16/26/38/50.

- Move Speed Slow increased from 10%/15%/20%/25% to 15%/20%/25%30%

- **New**: Arcane Curse now applies spell damage reduction 5/10/20/25%

- **New**: Casting a spell under the debuff now applies a maximum of 3 stacks increasing slow, spell damage reduction, and dps by 5 each. (Similar values to max slow and max damage under old silenced penalty multiplier)

### Glaives of Wisdom (Rework)

- INT to damage increased from 20%/40%/60%/80% to 30%/60%/80%/100%

- **New**: Now silences the affected target for 1.5 seconds on the 4th debuff. (Different from the shard mechanic which silences any target on the 4 attack)

- **New**: Now deals x2 damage to creeps

### Last Word (Rework)

- **New**: Silencer can now manually trigger last word during the enchantment period for 50% of the base damage (60/ 80/ 100/ 120) and silence duration values (0.5/1/1.5/2)

- Initial debuff duration (enchantment timer) reduced to 3 seconds

- **New**: Silence duration is re-scaled to: (1/2/3/4) if the debuff is triggered by the caster and : 2/4/6/7 if the enchantment timer is allowed to expire (This incentivizes a player to combo stuns with last word or punishes the opponent for not casting a spell when affected by the debuff)

- Cool down increased from 22/18/14/10 to 30/25/20/15

- Manacost increased from 100/105/110/115 to 110/115/120/125

### Aghanim's Shard (Rework)

- Causes Glaives of Wisdom to bounce within 500 AoE of the primary target

- **New**: Silenced targets from Glaives of Wisdom are now also affected by Arcane Curse

### Aghanim's Scepter (Rework)

- Arcane Curse becomes un-dispellable and Global Silence now applies the current level of Arcane Curse to all affected units

### Global Silence

- Cool down increased from 130/115/100 to 140/130/120

### General

- lvl 10 Talent +20 attack speed replaced with +125 attack range

- lvl 10 Talent +10 Arcane Curse dps replaced with +100 AoE Arcane Curse

- **New**: lvl 15 talent -20s Global Silence replaced with +2 INT stolen from any unit that dies under the effect Arcane Curse

- **New**: lvl 20 Talent +1 Glaives of Wisdom bounce replaced with +2 INT Stolen per kill. Applies retroactively.

- lvl 20 talent Arcane Curse un-dispellable replaced with +600 Last Word AoE

- **New**: lvl 25 Talnet +25% Glaives of Wisdom Damage replaced with Glaives of Wisdom pierce debuff immunity

**Comment by sloperun3az** [comment]
After perusing your most recent responses and experimenting with the <a href="https://poppyplaygame.io/">**poppy playtime**</a> game, I believe I now have a fundamental understanding of how it operates. I look forward to conducting additional tests to determine how I can make everything function. Many thanks to each of you!
